pkg/util/nvidia/paths.go

```diff
@@ -82,14 +82,19 @@ func nvliblist(nvidiaDir string) ([]string, error) {
 // Paths returns list of nvidia libraries and binaries that should
 // be added to mounted into container if it needs NVIDIA GPUs.
 func Paths(nvidiaDir string, envPath string) ([]string, []string, error) {
+	ldConfig, err := exec.LookPath("ldconfig")
+	if err != nil {
+		sylog.Debugf("ldconfig binary not found in %s, will search in %s", err, os.Getenv("PATH"), envPath)
```

Comment:
In this `Debugf` you have two `%s`, but three values. Should probably be changed to something like:

```suggestion
		sylog.Debugf("%v: ldconfig binary not found in %s, will search in %s", err, os.Getenv("PATH"), envPath)
```